test/test_mtg.py

```diff
@@ -0,0 +1,10 @@
+import openalea
+from openalea.deploy.shared_data import shared_data
+from openalea.mtg import MTG
+from oawidgets.mtg import plot
+
+
+def test_mtg():
+    data = shared_data(openalea.mtg)
```

Comment:
oups. shared_data changed to import openalea.mtg.data.
Perhaps add the file in oawidgets directly...